WooCommerce/Classes/POS/Presentation/ItemListView.swift

```diff
@@ -14,15 +14,15 @@ struct ItemListView: View {
         VStack {
             headerView
             switch viewModel.state {
-            case .empty(let emptyModel):
-                emptyView(emptyModel)
+            case .empty, .error:
+                // These cases are handled directly in the dashboard, we do not render
+                // a specific view within the ItemListView to handle them
+                EmptyView()
```

Comment:
Bit of a drive-by as I'm also looking at errors...

What's the benefit of handling the empty and error states in the Dashboard? 

I would expect that it was a cleaner separation of concerns for the ItemListView to show its own empty/error views.

I think it's because they are full screen, but you could delegate the fullscreen logic to the Dashboard view, while implementing the error/empty states here still.

Reply:
I started this trend with putting `PointOfSaleLoadingView()` on top of `PointOfSaleDashboardView`. 

I initially started by implementing it in ItemList but I found it hard to work with, since we expect this loading to appear above all the content, it was especially problematic UI-wise because of a previous version of a toolbar. However, maybe now it would be possible to implement it in `ItemListView` as well.

Reply:
> I think it's because they are full screen, but you could delegate the fullscreen logic to the Dashboard view, while implementing the error/empty states here still

That's a good point, as you mention my initial idea to move these to the dashboard was because of the full-screen error messages, but at the same time it makes more sense to keep them on the Item list as is where are generated and not mix responsibilities. 

I'll keep the issue open on https://github.com/woocommerce/woocommerce-ios/issues/13543 and we can dig deeper if needed once we've tackled other priorities.